selenium/automate_chrome.sh

```diff
@@ -19,9 +19,15 @@ if [ -f "$input" ]; then
     method="chrome/local"
 fi
 
-./build-dev.sh $method $browser_version true
+if [ "$tag" == "beta" -o "$tag" == "dev" ]; then
+    method_channel="$method/$tag"
+else
+    method_channel="$method/stable"
+fi
+
+./build-dev.sh $method_channel $browser_version true
 if [ "$method" == "chrome/apt" ]; then
-    ./build-dev.sh $method $browser_version false
+    ./build-dev.sh $method_channel $browser_version false
```

Comment:
Doesn't `build-dev` script runs twice here? once before the if statement and then inside if statement.

Reply:
This is original Selenoid code behaviour. Apparently it is for avoiding caching issues related to APT cache archives and base docker images between different local APT method builds.